UnitsNet.Tests/UnitsNet.Tests.csproj

```diff
@@ -50,7 +50,105 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs" Exclude="obj\**" />
+    <Compile Include="CustomCode\AccelerationTests.cs" />
```

Comment:
Make sure to revert this, project files should not be modified as they use wildcards to include .cs files.